lib/Components/logout_alert.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flood_mobile/Constants/app_color.dart';
+import 'package:flutter/material.dart';
+
+class LogOutAlert extends StatelessWidget {
+  const LogOutAlert({Key? key, @required this.logoutOnClick})
+      : assert(logoutOnClick != null),
+        super(key: key);
+  final VoidCallback? logoutOnClick;
+
+  @override
+  Widget build(BuildContext context) {
+    var deviceHeight = MediaQuery.of(context).size.height;
+    return AlertDialog(
+        backgroundColor: AppColor.secondaryColor,
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.all(Radius.circular(14)),
+        ),
+        contentPadding: EdgeInsets.all(20),
+        content: Text(
+          'Are you sure you want to\n Log out ?',
+          textAlign: TextAlign.center,
+          style: TextStyle(
+              color: Colors.white, fontWeight: FontWeight.bold, fontSize: 20),
+        ),
+        actionsAlignment: MainAxisAlignment.spaceEvenly,
```

Comment:
I am getting the following error for the "actionAlignment"

```The named parameter 'actionsAlignment' isn't defined.```

Reply:
https://github.com/flutter/flutter/pull/85260
This feature has been added to master. I update to flutter ver 2.5